src/app/build.json

```diff
@@ -10,7 +10,7 @@
   "discussionUrl": "https://github.com/cryostatio/cryostat3/discussions",
   "knownIssuesUrl": "https://github.com/cryostatio/cryostat3/issues",
   "fileIssueUrl": "https://github.com/cryostatio/cryostat3/issues/new?labels=user+report,bug&body=Affects+__REPLACE_VERSION__",
-  "mailingListName": "Google Groups",
+  "mailingListName": "Google groups",
```

Comment:
I think this should be left as `Google Groups` since that's the title of the website it's linking to.